ucp/_libs/send_recv.pyx

```diff
@@ -127,25 +128,29 @@ def tag_recv(uintptr_t ucp_worker, buffer, size_t nbytes,
     cdef ucp_worker_h worker = <ucp_worker_h>ucp_worker
     cdef void *data = <void*><uintptr_t>(get_buffer_data(buffer,
                                          check_writable=True))
+    cdef ucp_tag_recv_callback_t cb = (
+        <ucp_tag_recv_callback_t>_tag_recv_callback
+    )
     cdef ucs_status_ptr_t status = ucp_tag_recv_nb(worker,
                                                    data,
                                                    nbytes,
                                                    ucp_dt_make_contig(1),
                                                    tag,
                                                    -1,
-                                                   _tag_recv_callback)
+                                                   cb)
```

Comment:
This is extremely nit picky but i found the name `_tag_recv_callback` to be useful when reading the code.  What do you think about renaming the variable `_tag_recv_cb` and the same for the other callback funcs ?

Reply:
Sure. That seems fine.

FWIW the only reason this variable is here is we could not fit the cast in the line length limit. 😉

Reply:
Addressed in the latest commit. Thanks!

Reply:
Thanks @jakirkham